lib/deferrable.d.ts

```diff
@@ -34,10 +34,10 @@
  *
  */
 export interface AbstractDeferrableStatic {
-    new (): AbstractDeferrable
-    (): AbstractDeferrable
+    new (): Deferrable
+    (): Deferrable
 }
-export interface AbstractDeferrable {
+export interface Deferrable {
```

Comment:
Note: Although supported, shouldn't we make this a class and remove the call without `new`?